dev/imap-compose.yml

```diff
@@ -1,4 +1,4 @@
-# This file adapted from github.com/antespi/docker-imap-devel
+# test IMAP servers for devlopment
```

Comment:
seperated compose files into seperate files, only breaks link, instead resorting to host ports